operator/duties/committee.go

```diff
@@ -91,7 +97,10 @@ func (h *CommitteeHandler) buildCommitteeDuties(attDuties []*eth2apiv1.AttesterD
 	// This approach reduces contention and improves performance, as multiple individual calls would be slower.
 	vs := h.validatorProvider.SelfParticipatingValidators(epoch)
 	for _, v := range vs {
-		h.validatorCommitteeIDs[v.ValidatorIndex] = v.CommitteeID()
+		h.validatorCommitteeDutyMap[v.ValidatorIndex] = &committeeDuty{
+			id:          v.CommitteeID(),
+			operatorIDs: v.OperatorIDs(),
+		}
```

Comment:
Probably out of this PR scope but still:
Since this is loaded on each slot with committee duties, I don't see a point in storing this? If you would've loaded it each epoch it will make more sense I guess, but then you need to reload probably on indices  change or reorg. So maybe storing this doesn't add much value ( since managing is complicated ). right now I don't see if this map is every evacuated for example. (validator removed, etc..)